routes/index.js

```diff
@@ -1,9 +1,9 @@
-var express = require('express');
+var express = require("express");
```

Comment:
```suggestion
const express = require("express");
```
`var` 에서 `const` 로 변경해주시면 더 좋을 것 같습니다.